framework/src/Volo.Abp.Threading/Volo/Abp/Linq/AsyncQueryableExecuter.cs

```diff
@@ -5,21 +5,30 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Volo.Abp.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
 
 namespace Volo.Abp.Linq
 {
     public class AsyncQueryableExecuter : IAsyncQueryableExecuter, ITransientDependency
     {
         protected IEnumerable<IAsyncQueryableProvider> Providers { get; }
+        public ILogger<AsyncQueryableExecuter> Logger { get; set; }
 
         public AsyncQueryableExecuter(IEnumerable<IAsyncQueryableProvider> providers)
         {
             Providers = providers;
+            Logger = NullLogger<AsyncQueryableExecuter>.Instance;
         }
+        
 
         protected virtual IAsyncQueryableProvider FindProvider<T>(IQueryable<T> queryable)
         {
-            return Providers.FirstOrDefault(p => p.CanExecute(queryable));
+            var provider = Providers.FirstOrDefault(p => p.CanExecute(queryable));
+            if (provider == null)
+                Logger.LogWarning("IAsyncQueryableProvider is null");
```

Comment:
We follow [Allman](https://en.wikipedia.org/wiki/Indentation_style#Allman_style) style. Please open curly brackets for if condition.